src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TemporaryAbsence.kt

```diff
@@ -14,8 +15,10 @@ data class TemporaryAbsence(
   val lastName: String,
   @Schema(description = "Date of birth", example = "1971-02-01")
   val dateOfBirth: LocalDate,
+  @JsonProperty("offenderNo")
```

Comment:
Think this is the model we return to the frontend? Need a transformation between the prison API representation and the welcome people into prison later.

We should be referring to this as prisonNumber in our code in the frontend